docusaurus/docs/React/custom-code-examples/channel-users-list.mdx

```diff
@@ -0,0 +1,112 @@
+---
+id: channel-user-lists
+sidebar_position: 2
+title: Channel Members and Online Status
+---
+
+In this example, we will demonstrate how to render the current channel members and their online status.
+
+## Render the Channel Members List
+
+Let's start the example by creating a simple members list component. To access the members list of the current channel, we will get the current channel using `useChannelStateContext` hook. 
+The example is a bit more convoluted, since we will add online presence updates at the next step.
+
+:::note
+In order for the client to receive updates for user presence, ensure that you are watching the channel with `channel.watch({ presence: true })`. More details can be found in the [&nearr; LLC documentation](https://getstream.io/chat/docs/javascript/watch_channel/?language=javascript).
+:::
+
+```tsx
+const Users = () => {
+  const { channel } = useChannelStateContext();
+  const [channelUsers, setChannelUsers] = useState<
+    Array<{ name: string; online: boolean }>
+  >([]);
+
+  useEffect(() => {
+    const updateChannelUsers = (e?: Event) => {
+      // test if the updated user is a member of this channel
+      if (!e || channel.state.members[e.user!.id] !== undefined) {
+        setChannelUsers(
+          Object.values(channel.state.members).map((user) => ({
+            name: user.user_id!,
+            online: !!user.user!.online,
+          }))
+        );
+      }
+    };
+
+    updateChannelUsers();
+  }, [client, channel]);
+
+  return (
+    <ul>
```

Comment:
let's add className='users-list' to the unordered list (more css comments below to complete this). All of our custom examples look really good and utilize CSS, helps with our credibility I think.